# boost installs an executable named b2, which clashes with the default name for
# this package. This package changes the executable name to avoid this clash.
diff --git a/setup.py b/setup.py
index f9c1bbe..76ca28f 100644
--- a/setup.py
+++ b/setup.py
@@ -135,6 +135,6 @@ setup(
     # "scripts" keyword. Entry points provide cross-platform support and allow
     # pip to create the appropriate form of executable for the target platform.
     entry_points={
-        'console_scripts': ['b2=b2.console_tool:main'],
+        'console_scripts': ['backblaze2=b2.console_tool:main'],
     },
 )
